usaspending_api/etl/management/commands/load_transactions_in_delta.py

```diff
@@ -221,6 +258,236 @@ def delete_records_sql(self):
+                            WHEN tn.type IN ('02', '03', '04', '05')  THEN 'grant'
+                            WHEN tn.type IN ('06', '10')              THEN 'direct payment'
+                            WHEN tn.type IN ('07', '08')              THEN 'loans'
+                            WHEN tn.type = '09'                       THEN 'insurance'
+                            WHEN tn.type = '11'                       THEN 'other'
+                            WHEN tn.type LIKE 'IDV%%'                 THEN 'idv'
+                            ELSE NULL
+                        END AS category,
+                        tn.action_date AS certified_date,
+                        CASE
+                            WHEN month(tn.action_date) > 9 THEN year(tn.action_date) + 1
+                            ELSE year(tn.action_date)
+                        END AS fiscal_year,
+                        tn.funding_agency_id,
+                        tn.unique_award_key AS generated_unique_award_id,
+                        tn.is_fpds,
+                        tn.last_modified_date,
+                        tn.id AS latest_transaction_id,
+                        tn.period_of_performance_current_end_date,
+                        tn.transaction_unique_id,
+                        tn.type,
+                        tn.type_description,
+                        -- FPDS Columns
+                        fpds.agency_id AS fpds_agency_id,
+                        fpds.referenced_idv_agency_iden AS fpds_parent_agency_id,
+                        fpds.parent_award_id AS parent_award_piid,
+                        fpds.piid,
+                        -- FABS Columns
+                        fabs.fain,
+                        fabs.uri,
+                        -- Excecutive Compensation
+                        -- This might not be correct. In award_helpers.py some extra CTE logic is being done to
+                        -- select executive compensation
+                        COALESCE(fpds.officer_1_amount, fabs.officer_1_amount) AS officer_1_amount,
+                        COALESCE(fpds.officer_1_name, fabs.officer_1_name)     AS officer_1_name,
+                        COALESCE(fpds.officer_2_amount, fabs.officer_2_amount) AS officer_2_amount,
+                        COALESCE(fpds.officer_2_name, fabs.officer_2_name)     AS officer_2_name,
+                        COALESCE(fpds.officer_3_amount, fabs.officer_3_amount) AS officer_3_amount,
+                        COALESCE(fpds.officer_3_name, fabs.officer_3_name)     AS officer_3_name,
+                        COALESCE(fpds.officer_4_amount, fabs.officer_4_amount) AS officer_4_amount,
+                        COALESCE(fpds.officer_4_name, fabs.officer_4_name)     AS officer_4_name,
+                        COALESCE(fpds.officer_5_amount, fabs.officer_5_amount) AS officer_5_amount,
+                        COALESCE(fpds.officer_5_name, fabs.officer_5_name)     AS officer_5_name,
```

Comment:
These Executive Compensation fields should be removed from the `transaction_latest` CTE now that you've created the `transaction_ec` CTE